packages/uhk-web/src/styles/common.scss

```diff
@@ -0,0 +1,10 @@
+%btn-link {
```

Comment:
I am not css/scss wizard but I like to use the classic cascading, so I prefer to use class selector instead of placeholder. And delete the extending from capture-keystroke-button.component.scss, macro-list.component.scss files

Reply:
@ert78gb thanks for the feedback I chose the placeholder scss feature as we need exactly the same styles in two totally different component.
Any better way of doing this without copy-paste all css class properties on both places?
_I like the class selector approach, but in this case I have to define a global one, is this what you want here?_